lib/workers/repository/onboarding/pr/index.spec.ts

```diff
@@ -63,6 +63,20 @@ describe('workers/repository/onboarding/pr/index', () => {
       ]);
     });
 
+    it('creates PR with footer and header', async () => {
+      await ensureOnboardingPr(
+        {
+          ...config,
+          prHeader: 'And this is a header for {{{repository}}}',
+          prFooter: 'And this is a footer for {{{repository}}}',
+        },
+        packageFiles,
+        branches
+      );
+      expect(platform.createPr).toHaveBeenCalledTimes(1);
+      expect(platform.createPr.mock.calls[0][0].prBody).toMatchSnapshot();
```

Comment:
The snapshot does not show what you perhaps think it should show? There's no repository in it, but it's probably because of incomplete test case

Reply:
I have update the test case